architecture/additional_concepts/sdn.adoc

```diff
@@ -105,13 +81,11 @@ the aforementioned registry so that the master allocates a subnet to the node.
 
 Next, OpenShift SDN creates and configures six network devices:
 
-* *br0*, the OVS bridge device that pod containers will be attached to. OpenShift
-SDN also configures a set of non-subnet-specific flow rules on this bridge.
-ifdef::openshift-origin[]
-The *multitenant* plug-in does this immediately.
-endif::[]
-The *ovssubnet* plug-in waits to do so until the SDN master announces the
-creation of the new node subnet.
+* *br0*, the OVS bridge device that pod containers will be attached to.
+OpenShift SDN also configures a set of non-subnet-specific flow rules on this
+bridge. The *ovs-multitenant* plug-in does this immediately. The *ovs-subnet*
+plug-in waits to do so until the SDN master announces the creation of the new
+node subnet.
```

Comment:
Actually, drop the last sentence here: the subnet plugin is being changed to do it immediately as well in git master, and even before then, the difference is negligible.